File: Codewars/Python/string_to_camel.py
def to_camel_case(text):#convert normal string to camel case

    # create two variables, one for result and one for flag
    result = ""
    flag = 0 

    for i in text:#iterate over string

        if i == "_" or i == "-": #if string is underscore or hypen then continue for next iteration and add 1 into flag
            flag = 1
            continue

        elif flag == 1: # if flag is 1 then uppercase the string and continue for nexr iteration and add 0 into flag

            result += i.upper()
            flag = 0
            continue

        result += i

    return result

print(to_camel_case("the-Stealth-Warrior"))

File: Coding_Platforms/Codewars/Golang/string_to_camel.go
package main

import (
	"fmt"
	"strings"
)

func main() {

	fmt.Println(ToCamelCase("The_Stealth_Warrior"))
}

func ToCamelCase(s string) string { // convert normal string to camel case

	// create two variables, one for result and one for flag
	result := ""
	upper := 0

	for _, i := range s { //iterate over string

		if i == 45 || i == 95 { //if string is underscore or hypen then continue for next iteration and add 1 into flag
			upper = 1
			continue

		} else if upper == 1 { //if flag is 1 then uppercase the string and continue for nexr iteration and add 0 into flag

			result += strings.ToUpper(string(i))
			upper = 0
			continue
		}

		result += string(i)

	}

	return result
}
